english/lego/lego.md

```diff
@@ -0,0 +1,17 @@
+The mallow man seems nice so you chat to him.
+
+He decides he likes you too, so he gets out his Lego's. He's got Star Wars, Lego Movie, space, technics, oh so much lego and you play with it all!
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.